lsst/ts/laser/csc.py

```diff
@@ -233,36 +282,43 @@ def change_wavelength(self,wavelength):
 
         Returns
         -------
+        None
 
         """
-        self._laser.MaxiOPG.set_wavelength(wavelength)
+        self._laser.MaxiOPG.change_wavelength(wavelength=wavelength)
+
+    def set_output_energy_level(self,output_energy_level):
+        self._laser.set_output_energy_level(output_energy_level)
 
     def run(self):
         """Propagates the laser.
 
         Returns
         -------
+        None
 
         """
-        self._laser.M_CPU800.set_propagate("ON")
+        self._laser.start_propagating()
 
     def stop(self):
         """Stops propagating the laser.
 
         Returns
         -------
+        None
 
         """
-        self._laser.M_CPU800.set_propagate("OFF")
+        self._laser.stop_propagating()
 
     def publish(self):
         """Updates the laser's attributes.
 
         Returns
         -------
+        None
 
         """
-        self._laser._publish()
+        self._laser.publish()
 
 
 class LaserDeveloperRemote:
```

Comment:
Not sure I understood the need for this class. Maybe replace it by a script that is part of the module?

Reply:
You mean the publish method?

Reply:
No the `LaserDeveloperRemote` class.